packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.base.tsx

```diff
@@ -1,34 +1,58 @@
 import * as React from 'react';
 import {
   BaseComponent,
-  css
+  classNamesFunction,
+  customizable,
+  IClassNames,
 } from '../../Utilities';
-import { IProgressIndicatorProps } from './ProgressIndicator.types';
-import * as stylesImport from './ProgressIndicator.scss';
-const styles: any = stylesImport;
+import {
+  IProgressIndicatorProps,
+  IProgressIndicatorStyleProps,
+  IProgressIndicatorStyles,
+} from './ProgressIndicator.types';
+
+const getClassNames = classNamesFunction<IProgressIndicatorStyleProps, IProgressIndicatorStyles>();
 
 // if the percentComplete is near 0, don't animate it.
 // This prevents animations on reset to 0 scenarios
 const ZERO_THRESHOLD = 0.01;
 
+/**
+* ProgressIndicator with no default styles.
+* [Use the `getStyles` API to add your own styles.](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Styling)
+*/
+@customizable('ProgressIndicator', ['theme'])
 export class ProgressIndicatorBase extends BaseComponent<IProgressIndicatorProps, {}> {
   public static defaultProps = {
     label: '',
     description: '',
     width: 180
   };
 
+  private _classNames: IClassNames<IProgressIndicatorStyles>;
+
   constructor(props: IProgressIndicatorProps) {
     super(props);
 
     this._warnDeprecations({
       title: 'label'
     });
 
+    this._classNames = getClassNames(this.props.getStyles, {
+      theme: this.props.theme!,
+      className: this.props.className,
+      barHeight: this.props.barHeight,
+      indeterminate: this.props.percentComplete === undefined ? true : false,
+    });
```

Comment:
caching this in the contructor means if classname or barheight change after render, we wouldn't update.